app/role_matchers.py

```diff
@@ -66,9 +66,9 @@ def invalid_role_id(role_id):
 
 def download_permission(role_id):
     if download_permission_regex.match(role_id):
-        return True
+        return True 
```

Comment:
Always be on the lookout for if statements where the branches only differ by a boolean.
You can just `return download_permission_regex.match(role_id)` instead of using an if-statement.

Reply:
👌